src/components/__tests__/MenuItem.spec.tsx

```diff
@@ -35,26 +33,32 @@ describe('Components', () => {
         title: 'Menu item title'
       }
```

Comment:
personally I prefer having a render function in tests instead of overriding the default props inside the tests. This also avoids having to change all tests if you e.g. have to wrap the component into a themeProvider and stuff. But as said only personal preference